### PR TITLE
Feature profile page

### PR DESCRIPTION
**Pull request by jhcruz**

### CODE REVIEW

File: src/pages/user-edit/user-edit.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+})
+export class UserEditPage {
+
+  constructor(public navCtrl: NavController, public navParams: NavParams) {}
+
+  ionViewDidLoad() {
+
+    console.log('ionViewDidLoad UserEditPage');
+  }
+  enviar(Otro: Object){
```
Comment (rodrigo-cervone) [inline src/pages/user-edit/user-edit.ts]: english pls

**Comment by rodrigo-cervone** [comment]
@jhcruz Please, review the conflicts in this PR.

**Comment by jhcruz** [comment]
Conflicts resolved.

File: src/models/user.model.ts [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
+    public avatar_url: string,
+    public public_repos: number,
+    public public_gists: number,
+    public followers: number,
+    public following: number,
+    public repos_url: string
+  ) {  }
+}
+
+export interface User extends UserEntity{
```
Comment (rodrigo-cervone) [inline src/models/user.model.ts]: Why are you extending an interface from a class? Can't you just add the missing properties in the User and use that? It looks, at least, weird.

File: src/pages/user-details/user-details.ts [hunk 3]
```diff
@@ -23,17 +25,15 @@ export class UserDetailsPage {
...
-    this.githubUsers.loadDetails(this.login).subscribe((user: User) => {
-      this.user = user;
-      this.logger.debug("User: ", this.user);
-    });
+    this.user=this.sessionService.userReturn;
   }
 
   ionViewDidLoad() {
-    // console.log('ionViewDidLoad UserDetailsPage');
+      //Nothing
```
Comment (rodrigo-cervone) [inline src/pages/user-details/user-details.ts]: Please, just remove the method.

File: src/pages/user-edit/user-edit.ts [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    private githubUsers: GithubUsers,
+    private logger: Logger,
+    public sessionService: SessionService
+
+  ) {
+    this.user = this.sessionService.userReturn;
+
+  }
+
+  ionViewDidLoad() {
```
Comment (rodrigo-cervone) [inline src/pages/user-edit/user-edit.ts]: Same here. Just remove the method.

File: src/pages/user-edit/user-edit.ts [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    // Nothing
+  }
+  /**
+  * Method to send th entity User to API of GitHub
+  * @Method
+  * @name send|
+  * @author {} Jaime Hernández
+  */
+  send(){
```
Comment (rodrigo-cervone) [inline src/pages/user-edit/user-edit.ts]: Please, add a @TODO here with the intended behavior.

File: src/pages/user-login/user-login.ts [hunk 6]
```diff
@@ -24,14 +25,18 @@ import { UserDetailsPage } from '../user-details/user-details';
 export class UserLoginPage {
 
   private userInfo: SigninInfo;
+  //public sessionService: SessionService;
```
Comment (rodrigo-cervone) [inline src/pages/user-login/user-login.ts]: Remove this line, please.

**Comment by rodrigo-cervone** [comment]
Overall looking good. Just some minor comments.

**Review by rodrigo-cervone — CHANGES_REQUESTED** [review]

**Comment by rodrigo-cervone** [comment]
Looking good. Please resolve the conflicting files and we can merge it. Thanks for the changes.

**Pull request merged**